### DISCUSSION TITLE
Feature request: Job monitoring via healthchecks.io

### DISCUSSION THREAD

**Discussion by mrclschstr** [PremoWeb/chadburn#58] [Ideas]
It would be very handy to monitor scheduled jobs via [healthchecks.io](https://healthchecks.io/). I could image a global configuration like diun, for example (see: https://crazymax.dev/diun/config/watch/?h=healthchecks.io#healthchecks) or monitoring on a per-job basis.

What's your opinion about this?

**Comment by mpfl** [comment]
I'd also be interested in this.
